DOCUMENTATION.md

```diff
@@ -564,6 +565,31 @@ const client = await Client.newHazelcastClient(cfg);
+| hazelcast.client.shuffle.member.list                   | true                                | boolean         | The client shuffles the given member list to prevent all the clients to connect to the same member when this property is true. When it is set to false, the client tries to connect to the members in the given order.                                                                                                                                                                                              |
+| hazelcast.client.socket.no.delay                       | true                                | boolean         | The [setNoDelay](https://nodejs.org/api/net.html#net_socket_setnodelay_nodelay) setting for connections.                                                                                                                                                                                                                                                                                                            |
+| hazelcast.client.statistics.enabled                    | false                               | boolean         | If set to `true`, it enables collecting the client statistics and sending them to the cluster. When it is `true` you can monitor the clients that are connected to your Hazelcast cluster, using Hazelcast Management Center.                                                                                                                                                                                       |
+| hazelcast.client.statistics.period.seconds             | 3                                   | number          | Period in seconds the client statistics are collected and sent to the cluster.                                                                                                                                                                                                                                                                                                                                      |
+| hazelcast.discovery.public.ip.enabled                  | null(detection enabled)             | boolean or null | When set to `true`, the client will assume that it needs to use public IP addresses reported by members. When set to `false`, the client will always use private addresses reported by members.                                                                                                                                                                                                                     |
```

Comment:
I think we should document that, if it is null, we try to infer how the cluster is structured and act accordingly. I also think that, we should say that this inference is not %100 reliable and may result in false negatives (acting like `false`, while the cluster is set up to act like `true`)

Reply:
Why java client does not have null value but node.js client have?

Reply:
It also has that by default value

Reply:
Then I will open an issue since it is not documented. the property description in https://docs.hazelcast.com/imdg/4.2/clients/java.html#client-system-properties is not enough is it?